Tests/AsyncHTTPClientTests/HTTPClientTests.swift

```diff
@@ -61,23 +61,19 @@ class HTTPClientTests: XCTestCase {
         let response = try httpClient.get(url: "http://localhost:\(httpBin.port)/get").wait()
         XCTAssertEqual(.ok, response.status)
     }
-
-    func testGetWithSharedEventLoopGroup() throws {
```

Comment:
This shouldn't be needed anymore now that there isn't anymore the threading issues. This test could be re-introduced when we add a `requires` event loop preference to check that the requirements are always met.